spec/rspec/core/formatters_spec.rb

```diff
@@ -206,5 +206,14 @@ module RSpec::Core::Formatters
         end
       end
     end
+
+    describe "#change_output_stream" do
+      let(:new_output) { double 'output io' }
+      it "change output stream to all formatters" do
+        loader.add :documentation, output
+        expect(loader.formatters.first).to receive(:change_output).with(new_output)
+        loader.change_output_stream(new_output)
```

Comment:
I think it'd be nicer to use a `StringIO` here and assert that future notifications write into it.